### DISCUSSION TITLE
Support for "MIXED" string case

### DISCUSSION THREAD

**Discussion by YroslavNykolyshyn** [instancio/instancio#1050] [Ideas]
I haven't found a good way to handle string data generation for custom **MIXED** case.
I noticed that you recently added the ability to select the string type. 

> New setting **Keys.STRING_TYPE**
> The new setting allows customising generated string types via Settings:
> 
> ```
> Comment comment = Instancio.of(Comment.class)
>     .withSetting(Keys.STRING_TYPE, StringType.ALPHANUMERIC)
>     .create();
> ```
> or globally using instancio.properties:
> ```
> string.type=ALPHANUMERIC
> ```

Could you also add an option to customize the string case?
 New setting **Keys.STRING_CASE**
 
 ```
 Comment comment = Instancio.of(Comment.class)
     .withSetting(Keys.STRING_CASE, StringCase.MIXED)
     .create();
 ```
 or globally using instancio.properties:
 ```
 string.case=MIXED
 ```

**Comment by armandino** [comment]
@YroslavNykolyshyn thanks for the suggestion. Will be added in the next release.

- #1051
